2020-08-21 13:02:03.935 [error] <0.552.0> gen_server <0.552.0> terminated with reason: {#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 0,function => order_id,message => nil,module => nil,reason => nil},[{nil,order_id,[],[]},{'Elixir.ExchangeApiWeb.OrderLive','-render/1-fun-0-',2,[{file,"lib/exchange_api_web/live/order_live.html.leex"},{line,2}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},...]},...]}
2020-08-21 13:02:03.955 [error] <0.552.0> CRASH REPORT Process <0.552.0> with 0 neighbours crashed with reason: {#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 0,function => order_id,message => nil,module => nil,reason => nil},[{nil,order_id,[],[]},{'Elixir.ExchangeApiWeb.OrderLive','-render/1-fun-0-',2,[{file,"lib/exchange_api_web/live/order_live.html.leex"},{line,2}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},...]},...]}
2020-08-21 13:02:03.955 [error] <0.432.0> Supervisor {<0.432.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.552.0> exit with reason {#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 0,function => order_id,message => nil,module => nil,reason => nil},[{nil,order_id,[],[]},{'Elixir.ExchangeApiWeb.OrderLive','-render/1-fun-0-',2,[{file,"lib/exchange_api_web/live/order_live.html.leex"},{line,2}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},...]},...]} in context child_terminated
2020-08-21 13:05:56.720 [error] <0.590.0> CRASH REPORT Process <0.590.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,binary_to_integer,[<<"10ER039584">>],[]},{'Elixir.ExchangeApiWeb.OrderLive',mount,3,[{file,"lib/exchange_api_web/live/order_live.ex"},{line,9}]},{'Elixir.Phoenix.LiveView.Utils','maybe_call_mount!',3,[{file,"lib/phoenix_live_view/utils.ex"},{line,256}]},{'Elixir.Phoenix.LiveView.Static','call_mount_and_handle_params!',5,[{file,"lib/phoenix_live_view/static.ex"},{line,302}]},{'Elixir.Phoenix.LiveView.Static',...},...]},...},...},...}
2020-08-21 13:05:56.735 [error] <0.590.0> Cowboy stream 8 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.557.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,binary_to_integer,[<<"10ER039584">>],[]},{'Elixir.ExchangeApiWeb.OrderLive',mount,3,[{file,"lib/exchange_api_web/live/order_live.ex"},{line,9}]},{'Elixir.Phoenix.LiveView.Utils','maybe_call_mount!',3,[{file,"lib/phoenix_live_view/utils.ex"},{line,256}]},{'Elixir.Phoenix.LiveView.Static','call_mount_and_handle_params!',5,[{file,"lib/phoenix_live_view/static.ex"},{line,302}]},{'Elixir.Phoenix.LiveView.Static',...},...]},...},...}
2020-08-21 14:21:15.138 [error] <0.693.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 gen_server <0.693.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:21:15.167 [error] <0.693.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 CRASH REPORT Process <0.693.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:21:15.167 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.693.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23 in context child_terminated
2020-08-21 14:21:19.497 [error] <0.717.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 gen_server <0.717.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:21:19.498 [error] <0.717.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 CRASH REPORT Process <0.717.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:21:19.499 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.717.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23 in context child_terminated
2020-08-21 14:22:13.354 [error] <0.820.0> CRASH REPORT Process <0.820.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.KeyError',key => side,message => nil,term => #{'__struct__' => 'Elixir.Exchange.Trade',acknowledged_at => 1598016130092760000,buy_init_size => 6500,buy_order_id => <<"27">>,buyer_id => <<"1121">>,currency => 'GBP',price => 1001,sell_init_size => 6500,sell_order_id => <<"26">>,seller_id => <<"1121">>,size => 6500,ticker => 'AUXLND',trade_id => <<"514cd42a-e3b1-11ea-940f-88e9fe50bed2">>,type => full_fill}},[{'Elixir.ExchangeApiWeb.CompletedTradesLive',...},...]},...},...},...}
2020-08-21 14:22:13.365 [error] <0.820.0> Cowboy stream 17 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.627.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.KeyError',key => side,message => nil,term => #{'__struct__' => 'Elixir.Exchange.Trade',acknowledged_at => 1598016130092760000,buy_init_size => 6500,buy_order_id => <<"27">>,buyer_id => <<"1121">>,currency => 'GBP',price => 1001,sell_init_size => 6500,sell_order_id => <<"26">>,seller_id => <<"1121">>,size => 6500,ticker => 'AUXLND',trade_id => <<"514cd42a-e3b1-11ea-940f-88e9fe50bed2">>,type => full_fill}},[{'Elixir.ExchangeApiWeb.CompletedTradesLive',...},...]},...},...}
2020-08-21 14:22:22.058 [error] <0.837.0> CRASH REPORT Process <0.837.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.KeyError',key => side,message => nil,term => #{'__struct__' => 'Elixir.Exchange.Trade',acknowledged_at => 1598016130092760000,buy_init_size => 6500,buy_order_id => <<"27">>,buyer_id => <<"1121">>,currency => 'GBP',price => 1001,sell_init_size => 6500,sell_order_id => <<"26">>,seller_id => <<"1121">>,size => 6500,ticker => 'AUXLND',trade_id => <<"514cd42a-e3b1-11ea-940f-88e9fe50bed2">>,type => full_fill}},[{'Elixir.ExchangeApiWeb.CompletedTradesLive',...},...]},...},...},...}
2020-08-21 14:22:22.058 [error] <0.837.0> Cowboy stream 20 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.627.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.KeyError',key => side,message => nil,term => #{'__struct__' => 'Elixir.Exchange.Trade',acknowledged_at => 1598016130092760000,buy_init_size => 6500,buy_order_id => <<"27">>,buyer_id => <<"1121">>,currency => 'GBP',price => 1001,sell_init_size => 6500,sell_order_id => <<"26">>,seller_id => <<"1121">>,size => 6500,ticker => 'AUXLND',trade_id => <<"514cd42a-e3b1-11ea-940f-88e9fe50bed2">>,type => full_fill}},[{'Elixir.ExchangeApiWeb.CompletedTradesLive',...},...]},...},...}
2020-08-21 14:22:24.648 [error] <0.854.0> CRASH REPORT Process <0.854.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.KeyError',key => side,message => nil,term => #{'__struct__' => 'Elixir.Exchange.Trade',acknowledged_at => 1598016130092760000,buy_init_size => 6500,buy_order_id => <<"27">>,buyer_id => <<"1121">>,currency => 'GBP',price => 1001,sell_init_size => 6500,sell_order_id => <<"26">>,seller_id => <<"1121">>,size => 6500,ticker => 'AUXLND',trade_id => <<"514cd42a-e3b1-11ea-940f-88e9fe50bed2">>,type => full_fill}},[{'Elixir.ExchangeApiWeb.CompletedTradesLive',...},...]},...},...},...}
2020-08-21 14:22:24.649 [error] <0.854.0> Cowboy stream 23 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.627.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.KeyError',key => side,message => nil,term => #{'__struct__' => 'Elixir.Exchange.Trade',acknowledged_at => 1598016130092760000,buy_init_size => 6500,buy_order_id => <<"27">>,buyer_id => <<"1121">>,currency => 'GBP',price => 1001,sell_init_size => 6500,sell_order_id => <<"26">>,seller_id => <<"1121">>,size => 6500,ticker => 'AUXLND',trade_id => <<"514cd42a-e3b1-11ea-940f-88e9fe50bed2">>,type => full_fill}},[{'Elixir.ExchangeApiWeb.CompletedTradesLive',...},...]},...},...}
2020-08-21 14:23:04.404 [error] <0.862.0> CRASH REPORT Process <0.862.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.KeyError',key => side,message => nil,term => #{'__struct__' => 'Elixir.Exchange.Trade',acknowledged_at => 1598016130092760000,buy_init_size => 6500,buy_order_id => <<"27">>,buyer_id => <<"1121">>,currency => 'GBP',price => 1001,sell_init_size => 6500,sell_order_id => <<"26">>,seller_id => <<"1121">>,size => 6500,ticker => 'AUXLND',trade_id => <<"514cd42a-e3b1-11ea-940f-88e9fe50bed2">>,type => full_fill}},[{'Elixir.ExchangeApiWeb.CompletedTradesLive',...},...]},...},...},...}
2020-08-21 14:23:04.405 [error] <0.862.0> Cowboy stream 25 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.627.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.KeyError',key => side,message => nil,term => #{'__struct__' => 'Elixir.Exchange.Trade',acknowledged_at => 1598016130092760000,buy_init_size => 6500,buy_order_id => <<"27">>,buyer_id => <<"1121">>,currency => 'GBP',price => 1001,sell_init_size => 6500,sell_order_id => <<"26">>,seller_id => <<"1121">>,size => 6500,ticker => 'AUXLND',trade_id => <<"514cd42a-e3b1-11ea-940f-88e9fe50bed2">>,type => full_fill}},[{'Elixir.ExchangeApiWeb.CompletedTradesLive',...},...]},...},...}
2020-08-21 14:23:22.388 [error] <0.564.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.564.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:22.411 [error] <0.564.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.564.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:22.412 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.564.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:23:26.075 [error] <0.570.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.570.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:26.075 [error] <0.570.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.570.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:26.077 [error] <0.434.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.434.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.570.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:23:29.221 [error] <0.571.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.571.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:29.222 [error] <0.571.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.571.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:29.223 [error] <0.434.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.434.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.571.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:23:33.266 [error] <0.572.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.572.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:33.266 [error] <0.572.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.572.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:33.267 [error] <0.432.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.432.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.572.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:23:37.325 [error] <0.573.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.573.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:37.326 [error] <0.573.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.573.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:37.327 [error] <0.433.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.433.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.573.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:23:40.984 [error] <0.574.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.574.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:40.984 [error] <0.574.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.574.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:40.985 [error] <0.432.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.432.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.574.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:23:44.983 [error] <0.575.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.575.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:44.984 [error] <0.575.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.575.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:44.984 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.575.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:23:48.371 [error] <0.576.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.576.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:48.372 [error] <0.576.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.576.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:48.372 [error] <0.434.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.434.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.576.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:23:51.983 [error] <0.577.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.577.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:51.983 [error] <0.577.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.577.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:51.983 [error] <0.433.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.433.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.577.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:23:55.982 [error] <0.578.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.578.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:55.984 [error] <0.578.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.578.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:55.984 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.578.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:23:59.990 [error] <0.579.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.579.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:59.991 [error] <0.579.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.579.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:23:59.992 [error] <0.432.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.432.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.579.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:24:03.398 [error] <0.580.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.580.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:03.398 [error] <0.580.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.580.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:03.398 [error] <0.432.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.432.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.580.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:24:07.215 [error] <0.581.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.581.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:07.215 [error] <0.581.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.581.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:07.216 [error] <0.433.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.433.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.581.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:24:11.289 [error] <0.582.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.582.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:11.290 [error] <0.582.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.582.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:11.291 [error] <0.432.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.432.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.582.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:24:15.260 [error] <0.583.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.583.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:15.261 [error] <0.583.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.583.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:15.261 [error] <0.433.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.433.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.583.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:24:18.288 [error] <0.584.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.584.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:18.288 [error] <0.584.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.584.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:18.289 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.584.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:24:21.320 [error] <0.585.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.585.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:21.320 [error] <0.585.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.585.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:21.321 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.585.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:24:25.192 [error] <0.586.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.586.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:25.192 [error] <0.586.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.586.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:25.193 [error] <0.433.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.433.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.586.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:24:28.263 [error] <0.587.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.587.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:28.266 [error] <0.587.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.587.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:28.267 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.587.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:24:31.985 [error] <0.588.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.588.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:31.986 [error] <0.588.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.588.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:31.986 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.588.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:24:35.469 [error] <0.589.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.589.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:35.469 [error] <0.589.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.589.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:35.470 [error] <0.434.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.434.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.589.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:24:39.065 [error] <0.590.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.590.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:39.066 [error] <0.590.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.590.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:39.067 [error] <0.434.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.434.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.590.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:24:42.417 [error] <0.591.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.591.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:42.417 [error] <0.591.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.591.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:42.418 [error] <0.434.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.434.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.591.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:24:45.988 [error] <0.592.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.592.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:45.989 [error] <0.592.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.592.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:45.989 [error] <0.434.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.434.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.592.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:24:48.995 [error] <0.593.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.593.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:48.996 [error] <0.593.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.593.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:48.997 [error] <0.432.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.432.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.593.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:24:52.090 [error] <0.594.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.594.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:52.091 [error] <0.594.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.594.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:52.091 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.594.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:24:55.415 [error] <0.595.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.595.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:55.416 [error] <0.595.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.595.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:55.416 [error] <0.434.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.434.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.595.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:24:59.117 [error] <0.596.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.596.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:59.118 [error] <0.596.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.596.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:24:59.119 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.596.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:25:02.380 [error] <0.597.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.597.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:02.381 [error] <0.597.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.597.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:02.382 [error] <0.433.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.433.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.597.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:25:06.265 [error] <0.598.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.598.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:06.267 [error] <0.598.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.598.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:06.267 [error] <0.433.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.433.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.598.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:25:09.316 [error] <0.599.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.599.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:09.318 [error] <0.599.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.599.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:09.318 [error] <0.433.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.433.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.599.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:25:13.298 [error] <0.600.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.600.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:13.299 [error] <0.600.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.600.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:13.299 [error] <0.432.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.432.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.600.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:25:16.317 [error] <0.601.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.601.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:16.318 [error] <0.601.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.601.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:16.319 [error] <0.434.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.434.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.601.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:25:20.286 [error] <0.603.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.603.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:20.286 [error] <0.603.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.603.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:20.286 [error] <0.432.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.432.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.603.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:25:23.317 [error] <0.604.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.604.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:23.319 [error] <0.604.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.604.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:23.320 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.604.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:25:27.269 [error] <0.605.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.605.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:27.270 [error] <0.605.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.605.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:27.271 [error] <0.434.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.434.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.605.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:25:31.550 [error] <0.618.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.618.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:31.550 [error] <0.618.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.618.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:31.551 [error] <0.433.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.433.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.618.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:25:35.110 [error] <0.623.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.623.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:35.111 [error] <0.623.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.623.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:35.112 [error] <0.433.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.433.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.623.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:25:39.413 [error] <0.624.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.624.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:39.413 [error] <0.624.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.624.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:39.414 [error] <0.432.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.432.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.624.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:25:43.117 [error] <0.625.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 gen_server <0.625.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:43.118 [error] <0.625.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 CRASH REPORT Process <0.625.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25
2020-08-21 14:25:43.118 [error] <0.434.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:25 Supervisor {<0.434.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.625.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 25 in context child_terminated
2020-08-21 14:25:46.709 [error] <0.647.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 gen_server <0.647.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:25:46.710 [error] <0.647.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 CRASH REPORT Process <0.647.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:25:46.710 [error] <0.433.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 Supervisor {<0.433.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.647.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23 in context child_terminated
2020-08-21 14:25:50.005 [error] <0.652.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 gen_server <0.652.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:25:50.006 [error] <0.652.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 CRASH REPORT Process <0.652.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:25:50.007 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.652.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23 in context child_terminated
2020-08-21 14:25:53.466 [error] <0.653.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 gen_server <0.653.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:25:53.466 [error] <0.653.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 CRASH REPORT Process <0.653.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:25:53.467 [error] <0.432.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 Supervisor {<0.432.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.653.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23 in context child_terminated
2020-08-21 14:25:57.070 [error] <0.654.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 gen_server <0.654.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:25:57.070 [error] <0.654.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 CRASH REPORT Process <0.654.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:25:57.071 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.654.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23 in context child_terminated
2020-08-21 14:26:00.986 [error] <0.655.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 gen_server <0.655.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:26:00.986 [error] <0.655.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 CRASH REPORT Process <0.655.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:26:00.986 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.655.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23 in context child_terminated
2020-08-21 14:26:05.464 [error] <0.656.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 gen_server <0.656.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:26:05.465 [error] <0.656.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 CRASH REPORT Process <0.656.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:26:05.465 [error] <0.432.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 Supervisor {<0.432.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.656.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23 in context child_terminated
2020-08-21 14:26:09.065 [error] <0.657.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 gen_server <0.657.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:26:09.066 [error] <0.657.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 CRASH REPORT Process <0.657.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:26:09.066 [error] <0.434.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 Supervisor {<0.434.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.657.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23 in context child_terminated
2020-08-21 14:26:13.412 [error] <0.658.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 gen_server <0.658.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:26:13.413 [error] <0.658.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 CRASH REPORT Process <0.658.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:26:13.414 [error] <0.434.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 Supervisor {<0.434.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.658.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23 in context child_terminated
2020-08-21 14:26:16.422 [error] <0.659.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 gen_server <0.659.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:26:16.423 [error] <0.659.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 CRASH REPORT Process <0.659.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:26:16.423 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.659.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23 in context child_terminated
2020-08-21 14:26:20.217 [error] <0.660.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 gen_server <0.660.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:26:20.218 [error] <0.660.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 CRASH REPORT Process <0.660.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:26:20.218 [error] <0.433.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 Supervisor {<0.433.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.660.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23 in context child_terminated
2020-08-21 14:26:23.225 [error] <0.661.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 gen_server <0.661.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:26:23.225 [error] <0.661.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 CRASH REPORT Process <0.661.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:26:23.226 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.661.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23 in context child_terminated
2020-08-21 14:26:26.280 [error] <0.662.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 gen_server <0.662.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:26:26.282 [error] <0.662.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 CRASH REPORT Process <0.662.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:26:26.282 [error] <0.432.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 Supervisor {<0.432.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.662.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23 in context child_terminated
2020-08-21 14:27:07.547 [error] <0.553.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 gen_server <0.553.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:27:07.570 [error] <0.553.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 CRASH REPORT Process <0.553.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:27:07.571 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.553.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23 in context child_terminated
2020-08-21 14:27:10.216 [error] <0.568.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 gen_server <0.568.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:27:10.216 [error] <0.568.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 CRASH REPORT Process <0.568.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:27:10.217 [error] <0.433.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 Supervisor {<0.433.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.568.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23 in context child_terminated
2020-08-21 14:27:14.343 [error] <0.573.0> gen_server <0.573.0> terminated with reason: {'module could not be loaded',[{'Elixir.ExchangeApiWeb.CompletedTradesLive',handle_info,[tick,#{'__struct__' => 'Elixir.Phoenix.LiveView.Socket',assigns => #{buy_orders => [],flash => #{},live_action => get,live_module => 'Elixir.ExchangeApiWeb.CompletedTradesLive',sell_orders => [],ticker => 'AUXLND'},changed => #{},'connected?' => true,endpoint => 'Elixir.ExchangeApiWeb.Endpoint',fingerprints => {333288228247057508812083641928175899510,#{2 => {190920045012708318913604452799244854984,#{}}}},...}],...},...]}
2020-08-21 14:27:14.367 [error] <0.573.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info CRASH REPORT Process <0.573.0> with 0 neighbours crashed with reason: call to undefined function 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info(tick, #{'__struct__' => 'Elixir.Phoenix.LiveView.Socket',assigns => #{buy_orders => [],flash => #{},live_action => ...,...},...})
2020-08-21 14:27:14.368 [error] <0.435.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info Supervisor {<0.435.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.573.0> exit with reason call to undefined function 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info(tick, #{'__struct__' => 'Elixir.Phoenix.LiveView.Socket',assigns => #{buy_orders => [],flash => #{},live_action => ...,...},...}) in context child_terminated
2020-08-21 14:27:16.851 [error] <0.593.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 gen_server <0.593.0> terminated with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:27:16.852 [error] <0.593.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 CRASH REPORT Process <0.593.0> with 0 neighbours crashed with reason: no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23
2020-08-21 14:27:16.852 [error] <0.434.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info:23 Supervisor {<0.434.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.593.0> exit with reason no match of right hand value {error,<<"Ticker does not exist">>} in 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info/2 line 23 in context child_terminated
2020-08-21 14:28:00.680 [error] <0.590.0> CRASH REPORT Process <0.590.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.KeyError',key => side,message => nil,term => #{'__struct__' => 'Elixir.Exchange.Trade',acknowledged_at => 1598016477963348000,buy_init_size => 6500,buy_order_id => <<"27">>,buyer_id => <<"1121">>,currency => 'GBP',price => 99999,sell_init_size => 6500,sell_order_id => <<"28">>,seller_id => <<"1121">>,size => 6500,ticker => 'AUXLND',trade_id => <<"20a5aaf8-e3b2-11ea-8d63-88e9fe50bed2">>,type => full_fill}},[{'Elixir.ExchangeApiWeb.CompletedTradesLive',...},...]},...},...},...}
2020-08-21 14:28:00.692 [error] <0.590.0> Cowboy stream 4 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.562.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.KeyError',key => side,message => nil,term => #{'__struct__' => 'Elixir.Exchange.Trade',acknowledged_at => 1598016477963348000,buy_init_size => 6500,buy_order_id => <<"27">>,buyer_id => <<"1121">>,currency => 'GBP',price => 99999,sell_init_size => 6500,sell_order_id => <<"28">>,seller_id => <<"1121">>,size => 6500,ticker => 'AUXLND',trade_id => <<"20a5aaf8-e3b2-11ea-8d63-88e9fe50bed2">>,type => full_fill}},[{'Elixir.ExchangeApiWeb.CompletedTradesLive',...},...]},...},...}
2020-08-21 15:12:00.859 [error] <0.597.0> CRASH REPORT Process <0.597.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.KeyError',key => try,message => nil,term => #{'__struct__' => 'Elixir.Exchange.Trade',acknowledged_at => 1598019114884771000,buy_init_size => 6500,buy_order_id => <<"29">>,buyer_id => <<"1121">>,currency => 'GBP',price => 1001,sell_init_size => 6500,sell_order_id => <<"28">>,seller_id => <<"1121">>,size => 6500,ticker => 'AUXLND',trade_id => <<"445ff1fa-e3b8-11ea-ba4e-88e9fe50bed2">>,type => full_fill}},[{'Elixir.ExchangeApiWeb.CompletedTradesLive',...},...]},...},...},...}
2020-08-21 15:12:00.877 [error] <0.597.0> Cowboy stream 4 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.561.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.KeyError',key => try,message => nil,term => #{'__struct__' => 'Elixir.Exchange.Trade',acknowledged_at => 1598019114884771000,buy_init_size => 6500,buy_order_id => <<"29">>,buyer_id => <<"1121">>,currency => 'GBP',price => 1001,sell_init_size => 6500,sell_order_id => <<"28">>,seller_id => <<"1121">>,size => 6500,ticker => 'AUXLND',trade_id => <<"445ff1fa-e3b8-11ea-ba4e-88e9fe50bed2">>,type => full_fill}},[{'Elixir.ExchangeApiWeb.CompletedTradesLive',...},...]},...},...}
2020-08-21 15:14:38.628 [error] <0.663.0> CRASH REPORT Process <0.663.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[{ok,[]},trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...},...}
2020-08-21 15:14:38.629 [error] <0.663.0> Cowboy stream 3 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.650.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[{ok,[]},trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...}
2020-08-21 15:17:36.116 [error] <0.614.0> gen_server <0.614.0> terminated with reason: {'module could not be loaded',[{'Elixir.ExchangeApiWeb.CompletedTradesLive',handle_info,[tick,#{'__struct__' => 'Elixir.Phoenix.LiveView.Socket',assigns => #{flash => #{},live_action => get,live_module => 'Elixir.ExchangeApiWeb.CompletedTradesLive',ticker => <<"AUXLND">>,trades => [#{'__struct__' => 'Elixir.Exchange.Trade',acknowledged_at => 1598019417170235000,buy_init_size => 6500,buy_order_id => <<"29">>,buyer_id => <<"1121">>,currency => 'GBP',price => 99999,sell_init_size => 6500,sell_order_id => ...,...}]},...}],...},...]}
2020-08-21 15:17:36.240 [error] <0.614.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info CRASH REPORT Process <0.614.0> with 0 neighbours crashed with reason: call to undefined function 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info(tick, #{'__struct__' => 'Elixir.Phoenix.LiveView.Socket',assigns => #{flash => #{},live_action => get,...},...})
2020-08-21 15:17:36.240 [error] <0.442.0>@Elixir.ExchangeApiWeb.CompletedTradesLive:handle_info Supervisor {<0.442.0>,'Elixir.Supervisor.Default'} had child undefined started with {'Elixir.Phoenix.LiveView.Channel',start_link,undefined} at <0.614.0> exit with reason call to undefined function 'Elixir.ExchangeApiWeb.CompletedTradesLive':handle_info(tick, #{'__struct__' => 'Elixir.Phoenix.LiveView.Socket',assigns => #{flash => #{},live_action => get,...},...}) in context child_terminated
2020-08-21 15:17:46.000 [error] <0.643.0> CRASH REPORT Process <0.643.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[{ok,[]},trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...},...}
2020-08-21 15:17:46.013 [error] <0.643.0> Cowboy stream 3 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.607.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[{ok,[]},trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...}
2020-08-21 15:19:26.010 [error] <0.652.0> CRASH REPORT Process <0.652.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[{ok,[]},trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...},...}
2020-08-21 15:19:26.011 [error] <0.652.0> Cowboy stream 1 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.651.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[{ok,[]},trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...}
2020-08-21 15:19:47.501 [error] <0.660.0> CRASH REPORT Process <0.660.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[{ok,[]},price,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,11}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...},...}
2020-08-21 15:19:47.503 [error] <0.660.0> Cowboy stream 4 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.651.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[{ok,[]},price,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,11}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...}
2020-08-21 15:19:50.767 [error] <0.677.0> CRASH REPORT Process <0.677.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[{ok,[]},price,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,11}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...},...}
2020-08-21 15:19:50.767 [error] <0.677.0> Cowboy stream 6 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.651.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[{ok,[]},price,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,11}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...}
2020-08-21 15:20:02.843 [error] <0.685.0> CRASH REPORT Process <0.685.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[{ok,[]},trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...},...}
2020-08-21 15:20:02.844 [error] <0.685.0> Cowboy stream 9 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.651.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[{ok,[]},trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...}
2020-08-21 15:20:20.610 [error] <0.702.0> CRASH REPORT Process <0.702.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[{ok,[]},trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...},...}
2020-08-21 15:20:20.613 [error] <0.702.0> Cowboy stream 11 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.651.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[{ok,[]},trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...}
2020-08-21 15:20:35.634 [error] <0.750.0> CRASH REPORT Process <0.750.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[[],trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...},...}
2020-08-21 15:20:35.635 [error] <0.750.0> Cowboy stream 17 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.651.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[[],trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},{'Elixir.Enum',...},...]},...},...}
2020-08-21 15:21:05.111 [error] <0.767.0> CRASH REPORT Process <0.767.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[[],trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...},...}
2020-08-21 15:21:05.112 [error] <0.767.0> Cowboy stream 19 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.651.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[[],trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},{'Elixir.Enum',...},...]},...},...}
2020-08-21 15:21:38.483 [error] <0.780.0> CRASH REPORT Process <0.780.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[[],trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...},...}
2020-08-21 15:21:38.485 [error] <0.780.0> Cowboy stream 22 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.651.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[[],trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},{'Elixir.Enum',...},...]},...},...}
2020-08-21 15:22:00.175 [error] <0.788.0> CRASH REPORT Process <0.788.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.Phoenix.Router.NoRouteError',conn => #{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Cowboy.Conn',#{bindings => #{},body_length => 0,cert => undefined,has_body => false,headers => #{<<"accept">> => <<"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9">>,<<"accept-encoding">> => <<"gzip, deflate, br">>,<<"accept-language">> => <<"en-US,en;q=0.9,p...">>,...},...}},...},...},...},...},...},...}
2020-08-21 15:22:00.176 [error] <0.788.0> Cowboy stream 25 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.651.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.Phoenix.Router.NoRouteError',conn => #{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Cowboy.Conn',#{bindings => #{},body_length => 0,cert => undefined,has_body => false,headers => #{<<"accept">> => <<"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9">>,<<"accept-encoding">> => <<"gzip, deflate, br">>,<<"accept-language">> => <<"en-US,en;q=0.9,pt;...">>,...},...}},...},...},...},...},...}
2020-08-21 15:22:07.284 [error] <0.805.0> CRASH REPORT Process <0.805.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[[],trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...},...}
2020-08-21 15:22:07.286 [error] <0.805.0> Cowboy stream 4 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.793.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[[],trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},{'Elixir.Enum',...},...]},...},...}
2020-08-21 15:22:20.542 [error] <0.822.0> CRASH REPORT Process <0.822.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[[],trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...},...}
2020-08-21 15:22:20.544 [error] <0.822.0> Cowboy stream 7 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.793.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[[],trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},{'Elixir.Enum',...},...]},...},...}
2020-08-21 15:23:55.145 [error] <0.831.0> CRASH REPORT Process <0.831.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[[],trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...},...}
2020-08-21 15:23:55.147 [error] <0.831.0> Cowboy stream 1 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.830.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[[],trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},{'Elixir.Enum',...},...]},...},...}
2020-08-21 15:24:03.516 [error] <0.838.0> CRASH REPORT Process <0.838.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 1,function => dump,message => nil,module => 'Elixir.UUID',reason => nil},[{'Elixir.UUID',dump,[<<"9d4691ce-e3b9-11ea-ab7a-88e9fe50bed2">>],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive',mount,3,[{file,"lib/exchange_api_web/live/completed_trade_live.ex"},{line,7}]},{'Elixir.Phoenix.LiveView.Utils','maybe_call_mount!',3,[{file,"lib/phoenix_live_view/utils.ex"},{line,256}]},{'Elixir.Phoenix.LiveView.Static',...},...]},...},...},...}
2020-08-21 15:24:03.527 [error] <0.838.0> Cowboy stream 3 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.830.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 1,function => dump,message => nil,module => 'Elixir.UUID',reason => nil},[{'Elixir.UUID',dump,[<<"9d4691ce-e3b9-11ea-ab7a-88e9fe50bed2">>],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive',mount,3,[{file,"lib/exchange_api_web/live/completed_trade_live.ex"},{line,7}]},{'Elixir.Phoenix.LiveView.Utils','maybe_call_mount!',3,[{file,"lib/phoenix_live_view/utils.ex"},{line,256}]},{'Elixir.Phoenix.LiveView.Static',...},...]},...},...}
2020-08-21 15:24:34.130 [error] <0.855.0> CRASH REPORT Process <0.855.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 1,function => dump,message => nil,module => 'Elixir.UUID',reason => nil},[{'Elixir.UUID',dump,[<<"9d4691ce-e3b9-11ea-ab7a-88e9fe50bed2">>],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive',mount,3,[{file,"lib/exchange_api_web/live/completed_trade_live.ex"},{line,7}]},{'Elixir.Phoenix.LiveView.Utils','maybe_call_mount!',3,[{file,"lib/phoenix_live_view/utils.ex"},{line,256}]},{'Elixir.Phoenix.LiveView.Static',...},...]},...},...},...}
2020-08-21 15:24:34.130 [error] <0.855.0> Cowboy stream 5 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.830.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 1,function => dump,message => nil,module => 'Elixir.UUID',reason => nil},[{'Elixir.UUID',dump,[<<"9d4691ce-e3b9-11ea-ab7a-88e9fe50bed2">>],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive',mount,3,[{file,"lib/exchange_api_web/live/completed_trade_live.ex"},{line,7}]},{'Elixir.Phoenix.LiveView.Utils','maybe_call_mount!',3,[{file,"lib/phoenix_live_view/utils.ex"},{line,256}]},{'Elixir.Phoenix.LiveView.Static',...},...]},...},...}
2020-08-21 15:24:57.336 [error] <0.552.0> CRASH REPORT Process <0.552.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[[],trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},...]},...},...},...}
2020-08-21 15:24:57.348 [error] <0.552.0> Cowboy stream 1 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.551.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"argument error">>},[{erlang,apply,[[],trade_id,[]],[]},{'Elixir.ExchangeApiWeb.CompletedTradeLive','-render/1-fun-1-',2,[{file,"lib/exchange_api_web/live/completed_trade_live.html.leex"},{line,3}]},{'Elixir.Phoenix.LiveView.Diff',traverse,5,[{file,"lib/phoenix_live_view/diff.ex"},{line,250}]},{'Elixir.Phoenix.LiveView.Diff','-traverse_dynamic/5-fun-0-',3,[{file,"lib/phoenix_live_view/diff.ex"},{line,321}]},{'Elixir.Enum',...},...]},...},...}
2020-08-21 15:57:04.367 [error] <0.751.0> CRASH REPORT Process <0.751.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"assign @trade not available in eex template.\n\nPlease make sure all proper assigns have been set. If this\nis a child template, ensure assigns are given explicitly by\nthe parent template as they are not automatically forwarded.\n\nAvailable assigns: [:flash, :live_action, :live_module, :socket, :ticker, :trade_id]\n">>},[{'Elixir.Phoenix.LiveView.Engine','fetch_assign!',2,[{file,"lib/phoenix_live_view/engine.e..."},...]},...]},...},...},...}
2020-08-21 15:57:04.384 [error] <0.751.0> Cowboy stream 12 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.659.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"assign @trade not available in eex template.\n\nPlease make sure all proper assigns have been set. If this\nis a child template, ensure assigns are given explicitly by\nthe parent template as they are not automatically forwarded.\n\nAvailable assigns: [:flash, :live_action, :live_module, :socket, :ticker, :trade_id]\n">>},[{'Elixir.Phoenix.LiveView.Engine','fetch_assign!',2,[{file,"lib/phoenix_live_view/engine.ex"},...]},...]},...},...}
2020-08-21 15:58:07.478 [error] <0.759.0> CRASH REPORT Process <0.759.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"assign @trade not available in eex template.\n\nPlease make sure all proper assigns have been set. If this\nis a child template, ensure assigns are given explicitly by\nthe parent template as they are not automatically forwarded.\n\nAvailable assigns: [:flash, :live_action, :live_module, :socket, :ticker, :trade_id]\n">>},[{'Elixir.Phoenix.LiveView.Engine','fetch_assign!',2,[{file,"lib/phoenix_live_view/engine.e..."},...]},...]},...},...},...}
2020-08-21 15:58:07.480 [error] <0.759.0> Cowboy stream 1 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.758.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"assign @trade not available in eex template.\n\nPlease make sure all proper assigns have been set. If this\nis a child template, ensure assigns are given explicitly by\nthe parent template as they are not automatically forwarded.\n\nAvailable assigns: [:flash, :live_action, :live_module, :socket, :ticker, :trade_id]\n">>},[{'Elixir.Phoenix.LiveView.Engine','fetch_assign!',2,[{file,"lib/phoenix_live_view/engine.ex"},...]},...]},...},...}
2020-08-21 15:58:10.453 [error] <0.776.0> CRASH REPORT Process <0.776.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"assign @trade not available in eex template.\n\nPlease make sure all proper assigns have been set. If this\nis a child template, ensure assigns are given explicitly by\nthe parent template as they are not automatically forwarded.\n\nAvailable assigns: [:flash, :live_action, :live_module, :socket, :ticker, :trade_id]\n">>},[{'Elixir.Phoenix.LiveView.Engine','fetch_assign!',2,[{file,"lib/phoenix_live_view/engine.e..."},...]},...]},...},...},...}
2020-08-21 15:58:10.454 [error] <0.776.0> Cowboy stream 3 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.758.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"assign @trade not available in eex template.\n\nPlease make sure all proper assigns have been set. If this\nis a child template, ensure assigns are given explicitly by\nthe parent template as they are not automatically forwarded.\n\nAvailable assigns: [:flash, :live_action, :live_module, :socket, :ticker, :trade_id]\n">>},[{'Elixir.Phoenix.LiveView.Engine','fetch_assign!',2,[{file,"lib/phoenix_live_view/engine.ex"},...]},...]},...},...}
2020-08-21 16:00:20.490 [error] <0.787.0> CRASH REPORT Process <0.787.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"assign @trade not available in eex template.\n\nPlease make sure all proper assigns have been set. If this\nis a child template, ensure assigns are given explicitly by\nthe parent template as they are not automatically forwarded.\n\nAvailable assigns: [:flash, :live_action, :live_module, :socket, :ticker, :trade_id]\n">>},[{'Elixir.Phoenix.LiveView.Engine','fetch_assign!',2,[{file,"lib/phoenix_live_view/engine.e..."},...]},...]},...},...},...}
2020-08-21 16:00:20.491 [error] <0.787.0> Cowboy stream 1 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.786.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"assign @trade not available in eex template.\n\nPlease make sure all proper assigns have been set. If this\nis a child template, ensure assigns are given explicitly by\nthe parent template as they are not automatically forwarded.\n\nAvailable assigns: [:flash, :live_action, :live_module, :socket, :ticker, :trade_id]\n">>},[{'Elixir.Phoenix.LiveView.Engine','fetch_assign!',2,[{file,"lib/phoenix_live_view/engine.ex"},...]},...]},...},...}
2020-08-21 16:00:24.954 [error] <0.804.0> CRASH REPORT Process <0.804.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"assign @trade not available in eex template.\n\nPlease make sure all proper assigns have been set. If this\nis a child template, ensure assigns are given explicitly by\nthe parent template as they are not automatically forwarded.\n\nAvailable assigns: [:flash, :live_action, :live_module, :socket, :ticker, :trade_id]\n">>},[{'Elixir.Phoenix.LiveView.Engine','fetch_assign!',2,[{file,"lib/phoenix_live_view/engine.e..."},...]},...]},...},...},...}
2020-08-21 16:00:24.955 [error] <0.804.0> Cowboy stream 3 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.786.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"assign @trade not available in eex template.\n\nPlease make sure all proper assigns have been set. If this\nis a child template, ensure assigns are given explicitly by\nthe parent template as they are not automatically forwarded.\n\nAvailable assigns: [:flash, :live_action, :live_module, :socket, :ticker, :trade_id]\n">>},[{'Elixir.Phoenix.LiveView.Engine','fetch_assign!',2,[{file,"lib/phoenix_live_view/engine.ex"},...]},...]},...},...}
2020-08-21 16:03:24.400 [error] <0.814.0> CRASH REPORT Process <0.814.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"assign @trade not available in eex template.\n\nPlease make sure all proper assigns have been set. If this\nis a child template, ensure assigns are given explicitly by\nthe parent template as they are not automatically forwarded.\n\nAvailable assigns: [:flash, :live_action, :live_module, :socket, :ticker, :trade_id]\n">>},[{'Elixir.Phoenix.LiveView.Engine','fetch_assign!',2,[{file,"lib/phoenix_live_view/engine.e..."},...]},...]},...},...},...}
2020-08-21 16:03:24.401 [error] <0.814.0> Cowboy stream 1 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.813.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"assign @trade not available in eex template.\n\nPlease make sure all proper assigns have been set. If this\nis a child template, ensure assigns are given explicitly by\nthe parent template as they are not automatically forwarded.\n\nAvailable assigns: [:flash, :live_action, :live_module, :socket, :ticker, :trade_id]\n">>},[{'Elixir.Phoenix.LiveView.Engine','fetch_assign!',2,[{file,"lib/phoenix_live_view/engine.ex"},...]},...]},...},...}
2020-08-21 16:03:28.269 [error] <0.831.0> CRASH REPORT Process <0.831.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"assign @trade not available in eex template.\n\nPlease make sure all proper assigns have been set. If this\nis a child template, ensure assigns are given explicitly by\nthe parent template as they are not automatically forwarded.\n\nAvailable assigns: [:flash, :live_action, :live_module, :socket, :ticker, :trade_id]\n">>},[{'Elixir.Phoenix.LiveView.Engine','fetch_assign!',2,[{file,"lib/phoenix_live_view/engine.e..."},...]},...]},...},...},...}
2020-08-21 16:03:28.270 [error] <0.831.0> Cowboy stream 3 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.813.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"assign @trade not available in eex template.\n\nPlease make sure all proper assigns have been set. If this\nis a child template, ensure assigns are given explicitly by\nthe parent template as they are not automatically forwarded.\n\nAvailable assigns: [:flash, :live_action, :live_module, :socket, :ticker, :trade_id]\n">>},[{'Elixir.Phoenix.LiveView.Engine','fetch_assign!',2,[{file,"lib/phoenix_live_view/engine.ex"},...]},...]},...},...}
2020-08-21 16:04:22.714 [error] <0.870.0> CRASH REPORT Process <0.870.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"assign @trade not available in eex template.\n\nPlease make sure all proper assigns have been set. If this\nis a child template, ensure assigns are given explicitly by\nthe parent template as they are not automatically forwarded.\n\nAvailable assigns: [:flash, :live_action, :live_module, :socket, :ticker, :trade_id]\n">>},[{'Elixir.Phoenix.LiveView.Engine','fetch_assign!',2,[{file,"lib/phoenix_live_view/engine.e..."},...]},...]},...},...},...}
2020-08-21 16:04:22.714 [error] <0.870.0> Cowboy stream 10 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.813.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"assign @trade not available in eex template.\n\nPlease make sure all proper assigns have been set. If this\nis a child template, ensure assigns are given explicitly by\nthe parent template as they are not automatically forwarded.\n\nAvailable assigns: [:flash, :live_action, :live_module, :socket, :ticker, :trade_id]\n">>},[{'Elixir.Phoenix.LiveView.Engine','fetch_assign!',2,[{file,"lib/phoenix_live_view/engine.ex"},...]},...]},...},...}
2020-08-21 16:04:26.709 [error] <0.887.0> CRASH REPORT Process <0.887.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"assign @trade not available in eex template.\n\nPlease make sure all proper assigns have been set. If this\nis a child template, ensure assigns are given explicitly by\nthe parent template as they are not automatically forwarded.\n\nAvailable assigns: [:flash, :live_action, :live_module, :socket, :ticker, :trade_id]\n">>},[{'Elixir.Phoenix.LiveView.Engine','fetch_assign!',2,[{file,"lib/phoenix_live_view/engine.e..."},...]},...]},...},...},...}
2020-08-21 16:04:26.709 [error] <0.887.0> Cowboy stream 12 with ranch listener 'Elixir.ExchangeApiWeb.Endpoint.HTTP' and connection process <0.813.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"assign @trade not available in eex template.\n\nPlease make sure all proper assigns have been set. If this\nis a child template, ensure assigns are given explicitly by\nthe parent template as they are not automatically forwarded.\n\nAvailable assigns: [:flash, :live_action, :live_module, :socket, :ticker, :trade_id]\n">>},[{'Elixir.Phoenix.LiveView.Engine','fetch_assign!',2,[{file,"lib/phoenix_live_view/engine.ex"},...]},...]},...},...}
